CoolApp/CoolApp.xcodeproj/xcuserdata/niliev.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The `xcuserdata` directory should not be in source control. You'd better remove it and add it to `.gitignore`.

You can even go further and get all defaults from github's gitignore repo: https://github.com/github/gitignore/blob/master/Global/Xcode.gitignore.